src/graphnet/models/graphs/graph_definition.py

```diff
@@ -55,6 +59,12 @@ def __init__(
                                to None.
             seed: seed or Generator used to randomly sample perturbations.
                   Defaults to None.
+            add_inactive_sensors: If True, inactive sensors will be appended
+                to the graph with padded pulse information. Defaults to False.
+            sensor_mask: A list of sensor id's to be masked from the graph. Any
+                sensor listed here will be removed from the graph. Defaults to None.
+            string_mask: A list of string id's to be masked from the graph. Defaults to None.
+            sort_by: Name of node feature to sort by. Defaults to None.
```

Comment:
I'm not quite sure I understand what it means to have None as default for sort_by.

Reply:
It means that there is no sorting done.